web/modules/custom/collection/src/Form/JoinCollectionForm.php

```diff
@@ -22,6 +30,19 @@
  */
 class JoinCollectionForm extends FormBase {
 
+  /**
+   * CSS classes that will make a link look like a button.
+   */
+  const LINK_BUTTON_CLASSES = [
```

Comment:
These are theming specific classes that do not fit here I think. Shouldn't this be handled automatically by the form theming of buttons?